tests/third_party/cupy/manipulation_tests/test_rearrange.py

```diff
@@ -0,0 +1,76 @@
+import unittest
```

Comment:
please check `tests/third_party/cupy/manipulation_tests/test_transpose.py::TestTranspose::test_rollaxis_failure`, I believe we can remove `allow_fall_back_on_numpy` fixture there

Reply:
done